packages/browser-wallet/CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # Changelog
 
+## 0.7.2
+
+### Fixed
```

Comment:
When I try to connect I get an connection rejected, and it does not add  the site to connected.

Reply:
That happened because our atomWithStorage setters are actually async, but did this was not clear in the interface. I have updated the types and now the connection request popup awaits the setting of the updated connected sites before closing the window. Previously the window would close before it had the chance to update the storage.